src/components/calcite-slider/calcite-slider.scss

```diff
@@ -127,6 +127,17 @@ $tick-height: 4px;
   transform: translate(-7px, -8px);
 }
 :host([debug]) {
+  &:after {
+    content: "";
+    display: block;
+    width: calc(100% + 14px);
+    height: 50px;
+    position: absolute;
+    top: -16px;
+    left: -7px;
+    background-color: #ccc;
+    opacity: 0.3;
+  }
```

Comment:
Added this to aid in helping to make sure the labels stay within the gray box that is considered the "container" of the slider element.  Ends up looking like this when used:
![image](https://user-images.githubusercontent.com/821864/85436710-5365c600-b53e-11ea-850f-2a97ad4971fd.png)
![image](https://user-images.githubusercontent.com/821864/85436765-67a9c300-b53e-11ea-9de0-b108b35232a0.png)